dkan2.info.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
   - dkan_data
   - dkan_datastore
   - dkan_dummy_content
+  - dkan_json_form
```

Comment:
Are we happy with that module name? dkan_json_form?

Reply:
not unhappy with it... would dkan_dataset_form be better